RELEASE_PROCESS.md

```diff
@@ -25,4 +25,8 @@ The Testground team in general works on a two-week sprint schedule. We strive to
 
 At the end of every sprint, before the beginning of a new sprint, we do a `temperature check` and decide if we as a team are happy to create a release out of the work done during the sprint.
 
-If yes, we assign a `release manager` for the upcoming release, who is responsible for processing the checklist above and for creating a new release. During that period the rest of the team continues to work on outstanding tasks, or they move on to the next sprint. If necessary developers merge PRs into the `next` branch, which is synchronized and merged back to `master` after the `release manager` completes the release from the previous sprint. Alternatively at developers' discretion, folks are welcome to keep PRs approved and ready to be merged and avoid using the `next` branch.
+If yes, we assign a `release manager` for the upcoming release, who is responsible for processing the checklist above and for creating a new release.
+
+During that period the rest of the team continues to work on outstanding tasks, or they move on to the next sprint.
+
+If necessary developers merge PRs into the `next` branch, which is synchronized and merged back to `master` after the `release manager` completes the release from the previous sprint. Alternatively at developers' discretion, folks are welcome to keep PRs approved and ready to be merged, use `stacked PRs` and any other methods, and avoid using the `next` branch, while a release is underway.
```

Comment:
For the time being, I’d rather outline only 1 option for the question, “Where do I do my work?”. Having the dev merge their work into ‘Next’ removes the possibility that they may overlook a PR, once Master is ready again post-release.  It just seems like added complexity where mistakes could happen.  Only my 2c here.

Reply:
Let's assume that folks are smart enough to decide if they want to merge their changesets right away when this will make their lives easier, or if they are happy to merge them 1-2 days later (when the release it out) because they are not critical for example.

I really don't want to enforce only 1 option or rule, this is why the document says these are `guidelines` and also that managing work during the release period is at `developers discretion`.

Maintaining two branches has a cost as @coryschwartz said during the discussion, they should not be long-lived, so it is best if we evaluate this on a case-by-case basis.